cypress/integration/combobox.js

```diff
@@ -37,7 +37,6 @@ describe('combobox', () => {
       .should('have.value', 'Red')
   })
 
-  /* should be enabled if https://github.com/cypress-io/cypress/pull/3071 gets released.
```

Comment:
leave these commented please. We need cypress 3.2.0 for them. If you want to enable them please create another PR in which you also upgrade the cypress version and check they run locally as expected.

since our cypress tests don't run on CI anymore for a long time, maybe you can take a look at https://github.com/downshift-js/downshift/issues/704 and see if you have any experience in solving this. You will be a hero!

Reply:
Alright, I'll leave them as is for now and perhaps address them in a separate PR. I wasn't aware that the cypress tests aren't run on CI at the moment. I'll update the PR. I'll also see if I can do anything about #704, although I do not have much experience with cypress itself.